src/wcomms/HttpWorkerCzarModule.cc

```diff
@@ -241,7 +238,7 @@ json HttpWorkerCzarModule::_handleQueryStatus(std::string const& func) {
 
     auto const czInfo = wqsData->getCzInfo();
     LOGS(_log, LOG_LVL_TRACE, " HttpWorkerCzarModule::_handleQueryStatus req=" << jsReq.dump());
-    CzarIdType czId = czInfo->czId;
+    CzarId czId = czInfo->czId;
```

Comment:
Inconsistent naming convention everywhere. Should be:
```
CzarId const czarId = czarInfo->czarId;
```
Besides, why do you want to cache the value of `czarId` here instead of referring to `czarInfo->czarId` as needed?

Reply:
Because it is shorter and I find shorter lines of code easier to read. Same reason I prefer "qId" and "czId". It's clear what they mean and it makes the lines shorter. If I didn't feel it was clear what they meant, I would use the longer, more verbose versions.